src/plays/airbnb-experiences-clone/components/Card.jsx

```diff
@@ -0,0 +1,39 @@
+import katieZaferesImg from '../assets/images/katie-zaferes.png';
+import weddingImage from '../assets/images/wedding-photography.png';
+import mountainBikeImage from '../assets/images/mountain-bike.png';
+import starImage from '../assets/images/star.png';
+
+export default function Card(props) {
+  let badgeText;
+  if (props.openSpots === 0) {
+    badgeText = 'SOLD OUT';
+  } else if (props.location === 'Online') {
+    badgeText = 'ONLINE';
+  }
+
+  let cardImg;
+  if (props.coverImg === 'katie-zaferes.png') {
+    cardImg = katieZaferesImg;
+  } else if (props.coverImg === 'wedding-photography.png') {
+    cardImg = weddingImage;
+  } else if (props.coverImg === 'mountain-bike.png') {
+    cardImg = mountainBikeImage;
+  }
```

Comment:
We should use the state here to manage the component local data and update them based on the prop values.

Let's manage `badgeText` and `cardImg` with useState.

Reply:
Sounds good. I'd be happy to do so.

Reply:
Here, initially `badgeText` has no value so it will undefined I guess, when converting to state, adding an initial value might be helpful. As on if and else if conditions are there and no else condition.